nixos/modules/system/boot/systemd.nix

```diff
@@ -990,6 +992,7 @@ in
     systemd.targets.remote-fs.unitConfig.X-StopOnReconfiguration = true;
     systemd.targets.network-online.wantedBy = [ "multi-user.target" ];
     systemd.services.systemd-binfmt.wants = [ "proc-sys-fs-binfmt_misc.mount" ];
+    systemd.services.systemd-importd.environment = proxy_env;
```

Comment:
How does this work with other services that also need to go via the proxy?

Like `systemd-resolved` doing DNS over TCP through the proxy?

How do you configure it for other, non-systemd services?

Reply:
I'm afraid I won't be able to test this PR any more properly I've switched to a non-systemd based Linux distribution (still using Nix for some packages though ;). I could certainly add the setting for `systemd-resolved` as well and add a comment + newline. Do you know if `systemd-resolved` has support for HTTP proxies? Never tried DNS over TLS/HTTPS with `systemd-resolved`.

Reply:
I don't know TBH - quickly looking at it, it seems this uses openssl, not libcurl.

Let's only add it for importd for now - we can always add later.